lessons/module-3/unit-testing-react.md

```diff
@@ -499,11 +520,11 @@ class App extends Component {
     })
   }
 
-  groceryList = () => (
+  groceryList = () => {
```

Comment:
Your proposed change makes this incorrect, can you see why?

What is the function returning in the original version? What is it returning in your version?